app/views/collection/upload.html.slim

```diff
@@ -8,5 +8,5 @@
     .fglight
       p PNG, GIF, and JPG files are all acceptable.
       p Images should be oriented so that they are right-side-up.
-      p Images should be split down the spine, so that only one page is visible on an image.<br> (See the <a href="https://github.com/benwbrum/autosplit">autosplit</a> tool if you need to split two-page images.)
```

Comment:
Let's actually remove this whole paragraph, starting with the
```p Images should be split down the spine, so that only one page is visible on an image.```